### PR TITLE
docs: added project working

### LINKED ISSUE #3: [DOCS] Improve README
### Description

Improve the documentation. Add a bit more details about the project working

### Screenshots

_No response_

### PR DESCRIPTION
**Pull request by deeproop**
Closes #3 

## 👨‍💻 Changes proposed

Added the project working - 

- For the desired username entered information is fetched through Github API ( JavaScript fetch("URL") function is used ).
- Validity of user is checked.
- Then Data is converted to JSON and then filtered out.
- Customised JavaScript function is used to generate the output.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,5 +11,9 @@
 ## Project demo
 https://user-images.githubusercontent.com/51878265/184150017-651de573-d7f7-415f-b5bc-9caf28b7c6d8.mp4
 
-
+## Project Working1
```
Comment (Pradumnasaraf) [inline README.md]:
```suggestion
## Project Working
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Thank you for the PR, please fix the typo

**Comment by deeproop** [comment]
Typo Fixed 👍

**Review by MoKnowOrg — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
Looks good merge!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Pull request merged**